### PR TITLE
MEPTS-530: N5 - DSD Indicators

### PR DESCRIPTION
**Pull request by amosh101**

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/EriDSDCohortQueries.java [hunk 1]
```diff
@@ -1434,9 +1434,15 @@ private CohortDefinition getPatientsWhoCompletedGAAC() {
...
+   * or Continua (C) on Ficha Clinica – Master Card
+   *
+   * @return
+   */
   @DocumentedDefinition(
-      "Number of active patients on ART (Non-pregnant and Non-Breastfeeding not on TB treatment) who are in CA")
-  public CohortDefinition getPatientsWhoAreActiveAndParticipatingInAccessionClubs() {
+      "N5: Number of active patients on ART (Non-pregnant and Non-Breastfeeding not on TB treatment) who are in CA")
+  public CohortDefinition getN5() {
     CompositionCohortDefinition cd = new CompositionCohortDefinition();
```
[the remark this answers is not in the document]
Comment (amosh101) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/EriDSDCohortQueries.java:1444]: Done

[the remark this answers is not in the document]
Comment (psmatsinhe) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/EriDSDCohortQueries.java:1461]: @amosh101 I dont see any implementation of Criteria 3,4 and 5.

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/EriDSDCohortQueries.java [hunk 2]
```diff
@@ -1396,37 +1397,60 @@ private CohortDefinition getPatientsWhoCompletedGAAC() {
...
+        "breastfeeding",
+        EptsReportUtils.map(
+            breastfeeding, "onOrAfter=${endDate-18m},onOrBefore=${endDate},location=${location}"));
+    cd.addSearch(
+        "tbTreatment",
+        EptsReportUtils.map(
+            tbTreatment, "startDate=${startDate},endDate=${endDate},location=${location}"));
+
+    cd.setCompositionString(
+        "txCurr AND startOrContinueCA NOT (pregnant OR breastfeeding OR tbTreatment)");
```
Comment (psmatsinhe) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/EriDSDCohortQueries.java:1440]:
```suggestion
   cd.addSearch(
        "PregnantAndBreastfeedingAndOnTBTreatment",
        EptsReportUtils.map(
            getPregnantAndBreastfeedingAndOnTBTreatment(),
            "endDate=${endDate},location=${location}"));
    cd.setCompositionString(
        "txCurr AND startOrContinueCA NOT PregnantAndBreastfeedingAndOnTBTreatment");
```

Comment (psmatsinhe) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/EriDSDCohortQueries.java:1440]: @amosh101 please consider this

**Pull request merged**